src/Aspire.Hosting.Azure/Provisioning/Internal/DefaultProvisioningContextProvider.cs

```diff
@@ -217,14 +220,14 @@ public async Task<ProvisioningContext> CreateProvisioningContextAsync(JsonObject
             // Create a unique resource group name and save it in user secrets
```

Comment:
We never create a resource group via the ProvisioningContextProvider when in publish mode and always rely on it to be provisoned via the declaration in main.bicep.